File: CSharp-Part-1/5. Conditional Statements/ConditionalStatements/ExchangeIfGreater/ExchangeIfGreater.cs
// Write an if-statement that takes two double variables a and b and exchanges their values if the first one is greater than the second one. As a result print the values a and b, separated by a space.

using System;

class ExchangeIfGreater
{
    static void Main()
    {
        Console.WriteLine("Enter first number: ");
        double firstNumber = double.Parse(Console.ReadLine());
        Console.WriteLine("Enter second number: ");
        double secondNumber = double.Parse(Console.ReadLine());

        if (firstNumber > secondNumber)
        {
            double tempValue = firstNumber;
            firstNumber = secondNumber;
            secondNumber = tempValue;
        }
        Console.WriteLine("{0} {1}", firstNumber, secondNumber);
    }
}

File: JavaScript-Fundamentals/05. Conditional Statements/exchange-if-greater.js
/*
	Write an if statement that takes two double variables a and b and exchanges their values if the first one is greater than the second.
	As a result print the values a and b, separated by a space.
*/

var a = 5.5;
var b = 4.5;

function exchange() {
	if (a > b) {
		var tmp = a;
		a = b;
		b = tmp;
	}
}

exchange();
console.log(a + ' ' + b);
